multi-agent/adapters/inbound/flask/endpoints/resources.py

```
@@ -7,6 +7,61 @@
 
 resources_bp = Blueprint("resources", __name__)
 
+MAX_UPLOAD_SIZE_BYTES = 16384
+
+
+@resources_bp.route("/resource.upload-file", methods=["POST"])
+@with_authenticated_user
+def upload_resource_file(authenticated_user):
```

Ruff (0.15.20)

[warning] Unused function argument: `authenticated_user`

(ARG001)